error: 
    status - 500 
    message - SyntaxError: Unexpected string in JSON at position 32
    at JSON.parse (<anonymous>)
    at parse (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\body-parser\lib\types\json.js:89:19)
    at C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\body-parser\lib\read.js:128:18
    at AsyncResource.runInAsyncScope (node:async_hooks:202:9)
    at invokeCallback (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\raw-body\index.js:231:16)
    at done (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:539:35)
    at IncomingMessage.emit (node:domain:475:12)
    at endReadableNT (node:internal/streams/readable:1345:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) 
    url - /api/users 
    method - POST 
    IP - ::1
   {"timestamp":"2022-08-03T14:37:30.590Z"}
error: 
    status - 500 
    message - TypeError: console.error is not a constructor
    at _callee$ (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\src\services\/user.service.js:12:11)
    at tryCatch (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users 
    method - POST 
    IP - ::1
   {"timestamp":"2022-08-03T14:56:02.045Z"}
error: 
    status - 500 
    message - ReferenceError: error is not defined
    at _callee$ (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\src\services\/user.service.js:4:34)
    at tryCatch (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users 
    method - POST 
    IP - ::1
   {"timestamp":"2022-08-03T14:56:33.740Z"}
error: 
    status - 500 
    message - ValidationError: User validation failed: lastName: Path `lastName` is required., firstName: Path `firstName` is required.
    at model.Document.invalidate (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\mongoose\lib\document.js:2782:32)
    at C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\mongoose\lib\document.js:2574:17
    at C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\mongoose\lib\schematype.js:1273:9
    at processTicksAndRejections (node:internal/process/task_queues:78:11) 
    url - /api/v1/users 
    method - POST 
    IP - ::1
   {"timestamp":"2022-08-03T15:07:08.184Z"}
error: 
    status - 500 
    message - ValidationError: "firstName" is required 
    url - /api/v1/users 
    method - POST 
    IP - ::1
   {"timestamp":"2022-08-03T15:07:51.748Z"}
